README.md

```diff
@@ -22,14 +78,16 @@ Travix UI Components' repository.
    );
  }
  ```
-#### CSS
+##### CSS
 use file `node_modules/travix-ui-kit/dist/bundle.css`
 - you can create an alias in your webpack plugin
 - or inject it in your page current styles bundle
 - or copy to public folder and add as separate style file with `<link>`
 
 **Warning**: Directly using file `components/index.scss` not recommended. We're not promising that we will use SCSS in future or will keep file's structure
 
+---
+
 ## Living style guide
 
 ### Before installation
```

Comment:
I can not add comment on the exact line (96), since that part of the doc is not changed, but here we describe how developers of the travix-ui-kit can set a path to a different than the default theme.  This should be updated and also to be explained how to use the builder itself in this case. What i used during the testing was: `$ node ./builder -t ./themes/newTheme.yaml`.